test/proxy/run-plugin/runPlugin.t.sol

```diff
@@ -126,9 +126,10 @@ contract RunPlugin_Test is Proxy_Test {
 
         // Install the plugin and run it.
         registry.installPlugin(plugins.selfDestructer);
-        (bool success,) =
+        (bool success, bytes memory response) =
             address(proxy).call(abi.encodeWithSelector(plugins.selfDestructer.destroyMe.selector, users.bob));
-        success;
+        assertTrue(success);
+        assertEq(response.length, 0);
```

Comment:
Missing error messages.

Reply:
In the asserts, you mean?
Yes, my bad...